changelog.md

```diff
@@ -2,7 +2,10 @@
 
 ## UNRELEASED
 
-_Nothing yet!_
+**Added:**
+- react-native-bpk-component-card
+  - New component BpkCard
```

Comment:
Please follow the structure of previous "new component" changelogs

See https://github.com/Skyscanner/backpack/blob/master/changelog.md#2017-09-20---new-react-native-button-component:
```md
- react-native-bpk-component-text-input: 0.0.1 => 1.0.0
  - New component, see http://backpack.prod.aws.skyscnr.com/components/native/text-input
```

Reply:
Yes, not a problem, can I omit the docs link as it'll not be published by the time the PR will be merged?

Reply:
yeah sure

Reply:
Although obviously you won't yet have version numbers to include 😄

Reply:
so that leaves what I've put there? 😄 

```
+- react-native-bpk-component-card
+  - New component BpkCard
```

Reply:
...  i suppose it does! 😳 😂

Reply:
😆 😆 😆 😆

Reply:
I'll leave it like that then!